src/Page/Guide/View.elm

```diff
@@ -1,25 +1,26 @@
 module Page.Guide.View exposing (view)
 
-import Html.Styled exposing (Html, a, div, h1, li, p, text, ul)
+import Html.Styled as Html exposing (a, div, fromUnstyled, h1, li, p, text, ul)
```

Comment:
We could put the Html expose back in to avoid needing Html.Html in the type signatures, but not bothered.

Reply:
We might also be able to move the fromUnstyled into the renderer instead of having to use on every field? But not thought it through / tried it here. So if you did and got stuck, nevermind for now.

I have a general tactic that if I am calling a transform pattern in many places, I try to move it up a level, but that doesn't always work.